webpack.config.js

```diff
@@ -177,10 +178,11 @@ var config = {
         : path.join(__dirname, 'src/sentry/integration-docs/_platforms.json')
     },
     modules: [path.join(__dirname, staticPrefix), 'node_modules'],
-    extensions: ['*', '.jsx', '.js', '.json']
+    extensions: ['.less', '.jsx', '.js', '.json']
   },
   output: {
     path: distPath,
+    publicPath: '/_static/sentry/dist/',
```

Comment:
Are we using code splitting or anything here? I learned later this doesn't work and fixed it [in this PR](https://github.com/getsentry/sentry/pull/5623/commits/50b5ee383371741f64b835682c8e8faa63996b3d).